tests/phpunit/wp-tests-config.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * WordPress configuration file for PHPUnit tests.
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+// Test with WordPress debug mode (default).
+define( 'WP_DEBUG', true );
+
+// Set path to WordPress installation.
+define( 'ABSPATH', dirname( dirname( __DIR__ ) ) . '/vendor/roots/wordpress/' );
+
+/*
+ * This configuration file will be used by the copy of WordPress being tested.
+ * wordpress/wp-config.php will be ignored.
+ *
+ * WARNING WARNING WARNING!
+ * These tests will DROP ALL TABLES in the database with the prefix named below.
+ * DO NOT use a production database or one that is shared with something else.
+ */
+
+define( 'DB_NAME', 'wordpress_test' );
+define( 'DB_USER', 'root' );
+define( 'DB_PASSWORD', 'example' );
+define( 'DB_HOST', '127.0.0.1:9306' );
```

Comment:
It would be good to document that these values map to configuration of the `mysql` service in the E2E `bin/local-env/docker-compose.yml`, especially since they're fixed.